code/script.js

```diff
@@ -33,19 +47,166 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight;
 }
 
+// Here we have a function that "moves" the chat forward, 
+
+ const nextQuestion = (message) => {
+  console.log('questionNumber', questionNumber) 
+/* Here starts our if else statment, that lets the program know where we are in the chat,
+   example "if the question is equal to one do this, else move forward. It posts the user replay
+   that has the input value that they wrote. and after 1 second the next "bot" messages shows and it 
+   moves the chat forward.*/
+  if (questionNumber === 1) {
+    userReply(message)
+    input.value = ''
+    setTimeout(() => showHelpTypes(message), 1000)
+  } else if (questionNumber === 2) {
+    userReply(message)
+    setTimeout(() => showCodeMenu(message), 1000)
+  } else if (questionNumber === 3) {
+    userReply(message)
+    setTimeout(() => showCodeHelper(message), 1000)
+  } else if (questionNumber === 4) {
+    userReply(message)
+    setTimeout(() => showContactinformation(message), 1000)
+  } else {
+    userReply(message)
+    setTimeout(survey, 1000)
+  }
+}
+
 // Starts here
+/* here is the start of the chatbot (with question one, and now you're in the if statment) */
 const greeting = () => {
+  questionNumber = 1
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
+  botReply("Hello there, What's your name?")
   // Just to check it out, change 'bot' to 'user' here 👆
 }
+/* this shows after one second, and the questionnumber is now 2 (questionnumber++ gives this). The reply will include a message and the user input from previous question.
+  Also three buttons will show and the next question woill depend on which button is clicked.*/  
+const showHelpTypes = (msg) => {
```

Comment:
It's not abundandtly clear what the parameter `msg` represents here. An idea is to name it userName or something similar, as that seems to be the intended use.